### PR TITLE
Bugfix | V1-232 - Added UI changes

### PR DESCRIPTION
**Pull request by ostapdans99**
- Frontend

1. Added reusable MobileLink
2. Refactored style for text at Courses cards.
3. Refactored Similar Courses card UI
4. Fixed clickable zone for MenuIcon and click away for Notifications  at Header
5. Changed fontSizes for Profile info

### CODE REVIEW

File: frontend/src/components/Course/styled.ts [hunk 1]
```diff
@@ -122,6 +102,7 @@ export const ImageWrapper = styled('div')<Image>(({ imageUrl }) => {
 });
 
 export const CourseTitle = styled('p')<InfoContainerTypes>(({ type }) => ({
+  wordBreak: 'break-word',
   fontSize: '24px',
   fontWeight: 500,
   margin: '9px 5px 16px 0',
```
Comment (irina-sinitsyna) [inline frontend/src/components/Course/styled.ts:108]:
```suggestion
  margin: '9px 5px 16px 0',
  fontSize: '24px',
  fontWeight: 500,
  wordBreak: 'break-word',
```

File: frontend/src/components/Course/styled.ts [hunk 2]
```diff
@@ -161,6 +142,7 @@ export const CourseTitle = styled('p')<InfoContainerTypes>(({ type }) => ({
 
 export const CourseDescription = styled('p')<InfoContainerTypes>(
   ({ fontSize, lineHeight, type }) => ({
+    wordBreak: 'break-word',
     fontSize: '18px',
     lineHeight: '21px',
     padding: 0,
```
Comment (irina-sinitsyna) [inline frontend/src/components/Course/styled.ts:148]:
```suggestion
    padding: 0,
    fontSize: '18px',
    lineHeight: '21px',
    wordBreak: 'break-word',
```

File: frontend/src/components/Course/styled.ts (hunk 1 above)
Comment (ostapdans99) [reply frontend/src/components/Course/styled.ts]: done

File: frontend/src/components/Course/styled.ts (hunk 2 above)
Comment (ostapdans99) [reply frontend/src/components/Course/styled.ts]: done

**Review by irina-sinitsyna — APPROVED** [review]

**Review by zhuk-vezunchikz — APPROVED** [review]

**Pull request merged**